### PR TITLE
give priority to ELEMENTSD_EXE if exists

### PR DESCRIPTION
**Pull request by RCasatta**
along the lines https://github.com/shesek/electrumd/pull/3 and https://github.com/rust-bitcoin/bitcoind/pull/122

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -142,16 +138,20 @@ impl ElementsD {
...
+/// - if it's provided as a feature or as `ELEMENTSD_EXE` env var
+/// - if it's downloaded via a feature set
+/// - if it's in the path
 pub fn exe_path() -> anyhow::Result<String> {
-    match (downloaded_exe_path(), std::env::var("ELEMENTSD_EXE")) {
-        (Ok(_), Ok(_)) => Err(Error::BothFeatureAndEnvVar.into()),
-        (Ok(path), Err(_)) => Ok(path),
-        (Err(_), Ok(path)) => Ok(path),
-        (Err(_), Err(_)) => which::which("elementsd")
+    if let Ok(path) = std::env::var("ELECTRUMD_EXE") {
```
Comment (LeoComandini) [inline src/lib.rs]:
```suggestion
    if let Ok(path) = std::env::var("ELEMENTSD_EXE") {
```

**Comment by LeoComandini** [comment]
CI failures seems unrelated (checked few of them) not sure if we want to address them in this MR

**Comment by LeoComandini** [comment]
utACK 08a084079315087aaeca3e37d004b6d0a07409ac with comments, code review

File: src/lib.rs (hunk 1 above)
Comment (RCasatta) [reply src/lib.rs]: thanks!

**Pull request merged**